appengine/bigquery/app.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// This App Engine application uses its default service account to list all
+// the BigQuery datasets accessible via the BigQuery REST API.
+package sample
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/bigquery/v2"
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/urlfetch"
+)
+
+func init() {
+	http.HandleFunc("/", handle)
+}
+
+func handle(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+
+	// create a new App Engine context from the request.
+	c := appengine.NewContext(r)
```

Comment:
I think we want `c` -> `ctx`.

(`ctx` because we're using the actual `context` package now.)